Automattic/wp-calypso
# Framework: Upgrade qs to 6.5.1

Changelog: https://github.com/ljharb/qs/blob/master/CHANGELOG.md

Mostly to make the `sort` arg to `qs.stringify()` (introduced with v5.1.0) available for  https://github.com/Automattic/wp-calypso/pull/22593/files#r174065565

To test:
* This is all over the place, so probably a bit unwieldy to test; maybe just test a few occurrences (e.g. one for `decode`, one for `parse`, one for `stringify` -- see #23261) to gain confidence it works.
* Let's merge, deploy, and stick around in case things go 💥?
